examples/trimmer/airtun/Makefile

```diff
@@ -0,0 +1,67 @@
+AIRCRACK-NG=aircrack-ng
+AIRTUN-NG=airtun-ng
+AIRCRACK-NG_BITCODE=${AIRCRACK-NG}.bc
+AIRTUN-NG_BITCODE=${AIRTUN-NG}.bc
+AIRCRACK-NG_VERSION=1.1
+AIRCRACK-NG_NAME=${AIRCRACK-NG}-${AIRCRACK-NG_VERSION}
+AIRCRACK-NG_TAR=${AIRCRACK-NG_NAME}.tar.gz
+
+AIRCRACK-NG_MANIFEST=${AIRCRACK-NG}.manifest
+AIRTUN-NG_MANIFEST=${AIRTUN-NG}.manifest
+
+HELPER_SCRIPT=./build.sh
+HELPER_TAR=helper_scripts_and_files.tar.gz
+
+
+OUT_DIR?=trimmer
+LOG_FILE?=logs
+
+BITCODE_WRAPPER?=gclang
+BITCODE_EXTRACT?=get-bc
+OPT_NAME=opt-10
```

Comment:
Again use `opt`

Reply:
Done. changed opt-10 to opt for this example as well